src/lib/ops/validation.js

```diff
@@ -47,7 +47,7 @@ const users = {
     organizationId: validationRules.organizationId
   },
   createUser: {
-    id: validationRules.id.optional().allow('').description('User ID'),
+    id: validationRules.id.optional().description('User ID'),
```

Comment:
the string is validated only on the teams id, in the other cases was removed. Are you thinking at some validation that should be applied on IDs?